charts/otel-cloud-stack/charts/crds/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: v2
```

Comment:
this matches what's done in kube-prometheus-stack such that we can install the necessary prometheus CRDs in the right order. the installation of this is entirely optional and disabled by default.